mmv1/third_party/terraform/services/bigtable/resource_bigtable_gc_policy.go

```diff
@@ -56,6 +57,36 @@ func resourceBigtableGCPolicyCustomizeDiffFunc(diff tpgresource.TerraformResourc
 
 	return nil
 }
+func handleGCRulesDiffs(oldRules, newRules interface{}) bool {
```

Comment:
By convention, diff suppress funcs end in `DiffSuppress`. Please also add thorough unit tests for this function, similar to https://github.com/GoogleCloudPlatform/magic-modules/blob/ddb6b08f09cd83dd5c5c9ba82e670bcbea398bc0/mmv1/third_party/terraform/services/sql/resource_sql_database_internal_test.go#L7.

```suggestion
func gcRulesDiffSuppress(oldRules, newRules interface{}) bool {
```